### PR TITLE
refactor: improve wiki css

### PR DESCRIPTION
**Pull request by fraxken**

### CODE REVIEW

**Review by PierreDemailly — APPROVED** [review]

**Comment by fraxken** [comment]
fix #272 
fix #273

File: esbuild.config.js [hunk 1]
```diff
@@ -13,11 +13,13 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const kPublicDir = path.join(__dirname, "public");
 const kOutDir = path.join(__dirname, "dist");
 const kImagesDir = path.join(kPublicDir, "img");
+const kNodeModules = path.join(__dirname, "node_modules");
```
Comment (PierreDemailly) [inline esbuild.config.js:16]:
```suggestion
const kNodeModulesDir = path.join(__dirname, "node_modules");
```

More accurate based on above constants, wdyt ?

File: public/css/reset.css [hunk 2]
```diff
@@ -89,7 +89,7 @@ video {
   border: 0;
   font-size: 100%;
   font: inherit;
-  vertical-align: baseline;
+  /* vertical-align: baseline; */
```
Comment (PierreDemailly) [inline public/css/reset.css:92]:
```suggestion
```

To delete?

File: workspaces/documentation-ui/src/fetch.js [hunk 3]
```diff
@@ -55,7 +63,14 @@ export async function fetchNodeSecureWarningsByTitle(title, options = {}) {
...
   const httpResponse = await fetch(`${kRawGithubWarningsURL}/${title}.md`);
   const markdownResponse = await httpResponse.text();
-  const htmlResponse = kMarkdownConvertor.render(markdownResponse);
+  const htmlResponse = kMarkdownConvertor.render(
+    markdownResponse
+      .replaceAll("[!IMPORTANT]", "**IMPORTANT**")
+      .replaceAll("[!NOTE]", "**NOTE**")
+      .replaceAll("[!CAUTION]", "**CAUTION**")
+      .replaceAll("[!WARNING]", "**WARNING**")
+      .replaceAll("[!TIP]", "**TIP**")
```
Comment (PierreDemailly) [inline workspaces/documentation-ui/src/fetch.js:72]:
```suggestion
      .replaceAll("[!IMPORTANT]", "🚨")
      .replaceAll("[!NOTE]", "📝")
      .replaceAll("[!CAUTION]", "🔶")
      .replaceAll("[!WARNING]", "⚠️")
      .replaceAll("[!TIP]", "💡")
```

Maybe we could try to use emoji ? Just an idea

**Pull request merged**